src/PlatformServices/Services/ThreadOperations.cs

```diff
@@ -25,14 +24,43 @@ public class ThreadOperations : IThreadOperations
         /// <returns>Returns true if the action executed before the timeout. returns false otherwise.</returns>
         public bool Execute(Action action, int timeout, CancellationToken cancelToken)
         {
+#if NETSTANDARD1_4
+            var executionTask = Task.Factory.StartNew(action);
+            try
+            {
+                if (executionTask.Wait(timeout, cancelToken))
+                {
+                    return true;
+                }
+                else
+                {
+                    // Timed out.
+                    return false;
+                }
```

Comment:
```suggestion
                return executionTask.Wait(timeout, cancelToken);
```

Reply:
There was an abort logic inside else part. Kept the code because we need to decide what to do there. Depending or decision, I'll apply your suggestion or not :)